tokio-console/src/view/mod.rs

```diff
@@ -169,21 +174,24 @@ impl View {
                         update_kind = UpdateKind::Other;
                     }
                     key!(Enter) => {
-                        if let Some(op) = view.async_ops_table.selected_item().upgrade() {
-                            if let Some(task_id) = op.borrow().task_id() {
-                                let task = self
-                                    .tasks_list
-                                    .sorted_items
-                                    .iter()
-                                    .filter_map(|i| i.upgrade())
-                                    .find(|t| task_id == t.borrow().id());
+                        if let Some(op_weak) = view.async_ops_table.selected_item() {
+                            if let Some(op) = op_weak.upgrade() {
```

Comment:
As an alternative approach to changing `selected_item()` we could use `.and_then` here:

```suggestion
                        if let Some(op_weak) = view.async_ops_table.selected_item().and_then(|w| w.upgrade()) {
```

This removes the extra level of indentation.